civis/ml/tests/test_model.py

```diff
@@ -143,8 +143,8 @@ def foo(arg):
 
 @mock.patch.object(_model.cio, 'file_to_civis', return_value=-11)
 def test_stash_local_data_from_file(mock_file):
-    with tempfile.NamedTemporaryFile() as tempfname:
-        fname = tempfname.name
+    with TemporaryDirectory() as temp_dir:
+        fname = os.path.join(temp_dir, str(uuid.uuid4()))
```

Comment:
You don't need a uuid for anything created in a temporary directory -- the file will vanish as soon as you exit the context, and I don't see a danger of a name collision inside this test.